scripts/run-alphanet-parachain.sh

```diff
@@ -64,13 +77,13 @@ $PARACHAIN_BINARY \
     --ws-port $((PARACHAIN_PORT + 10 + 2)) \
     --validator \
     --name parachain_$PARACHAIN_INDEX \
-    --tmp \
+    $PARACHAIN_BASE_PATH \
     '-linfo,evm=trace,ethereum=trace,rpc=trace' \
     --chain $PARACHAIN_SPEC_PLAIN  \
     $PARACHAIN_BOOTNODES_ARGS \
     -- \
       --node-key ${PARACHAIN_KEYS[$PARACHAIN_INDEX]} \
-      --tmp \
+      $RELAY_BASE_PATH \
```

Comment:
It isn't necessary to pass separate base paths to the parachain node and the embedded relay chain node. I asked basti about this in element rococo room. Sounds like it might be harmless though?

> When running a parachain node, what if I give the embedded relay node and the parachain node different base paths? Example: `my-parachain --base-path /tmp/a -- --base-path /tmp/b` And what is the recommended practice for base path(s) in this situation?

>I think we override it, but there shouldn't happen anything special

Reply:
Ok I'll change it to the same